r2dbc-mysql/src/main/java/Bindings.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.jasync.r2dbc.mysql
+
+/**
+ * Statement parameters bindings.
+ */
+internal class Bindings {
+
+    private val bindings: MutableList<Map<Int, Any?>> = mutableListOf()
```

Comment:
how about calling it paramBindings?

Reply:
`bindings` lines nicely up with what we have in the other drivers as well.